CompetitionMARS/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,364 @@
+﻿
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using CompetitionMARS.Models;
+using CompetitionMARS.Pages;
+using CompetitionMARS.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using NUnit.Framework;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+namespace CompetitionMARS.Tests
+{
+    public class CertificationTests : CommonDriver
+    {
+        private Certification certPage;
+        public static ExtentReports extent;
+        public List<string> addedCertificationsEntries = new List<string>();
+
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            // Start the report
+            extent = ExtentManager.StartReport();
+
+            certPage = new Certification();
+            LoginActions loginobj = new LoginActions();
+            LoginData loginuser = loginobj.GetLoginData();
+
+            loginobj.Login(loginuser.Email, loginuser.Password);
+            certPage.NavigateToCertificationTab();
+            certPage.ClearAllCertification();
+        }
+        [Test, Order(1)]
+        public void CertificationTest_Valid()
+        {
+            ExtentTest test = extent.CreateTest("CertificationTest_Valid");
+
+            CertificationData cert = certPage.GetCertificationData("validdata1");
```

Comment:
you should not use 1 json file instead have a json file per test which will contain the data it needs instead of doing = certPage.GetCertificationData("validdata1"). the test should have the file where it converts json to objects here. Also but doing this its easier to maintain and also easy to read